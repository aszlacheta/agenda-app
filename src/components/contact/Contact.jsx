import React from 'react';
import './Contact.scss';

export default function Contact() {

    return <div className="contact">This page is not implemented yet...</div>
}