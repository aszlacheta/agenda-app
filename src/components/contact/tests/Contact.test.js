import React from 'react'
import renderer from 'react-test-renderer'
import Contact from '../Contact'

describe('Contact', () => {
  it('check if renders properly', () => {
    const tree = renderer
      .create(<Contact />)
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})
