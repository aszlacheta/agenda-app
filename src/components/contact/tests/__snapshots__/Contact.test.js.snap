// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Contact check if renders properly 1`] = `
<div
  className="contact"
>
  This page is not implemented yet...
</div>
`;
